src/sswsdk/dbconnector.py

```diff
@@ -9,6 +9,9 @@
 
 
 class SonicV1Connector(DBInterface):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
```

Comment:
Pavel is correct. `super(C, self)` is compatible with both Python 2 & 3. `super()` is available in Python 3 only.